keyboards/bluebell/swoop/keymaps/kyek/keymap_italian.h

```diff
@@ -0,0 +1,164 @@
+/* Copyright 2015-2016 Matthias Schmidtt
```

Comment:
This file is identical to the one in core and should be removed.

Reply:
Done 👍🏻